clang/docs/ReleaseNotes.rst

```diff
@@ -899,6 +899,7 @@ Arm and AArch64 Support
   * Arm Cortex-A520 (cortex-a520).
   * Arm Cortex-A720 (cortex-a720).
   * Arm Cortex-X4 (cortex-x4).
+  * Arm Cortex-M52 (cortex-m52).
```

Comment:
Is it worth splitting this list into one for -target=aarch64 and another for -target=arm?

Reply:
Sure. Something like this?

```
+  --target=arm
+     * Arm Cortex-M52 (cortex-m52).
+
+  --target=aarch64
+     * Arm Cortex-A520 (cortex-a520).
+     * Arm Cortex-A720 (cortex-a720).
+     * Arm Cortex-X4 (cortex-x4).
```